src/main/resources/schema/user.graphqls

```diff
@@ -0,0 +1,90 @@
+type User {
+  id: ID!
+  additionalSlot: Int
+  company: ID
+  countryCode: String
+  freetrialPeriod: String
+  purchasePeriod: String
+  purchaseType: String
+  tmpData: String
+  updateReason: String
+  nickname: String
+  officialFlag: Int
+  recommendType: Int
+}
+
+input UserFilter {
+  companyId: Int
+}
+
+input userRegisterInput {
```

Comment:
userRegisterInput sai -> UserRegisterInput
Check lại toàn bộ.

Reply:
fixed